frontend/src/pages/TermPlanner/DraggableCourse/DraggableCourse.jsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import { Draggable } from "react-beautiful-dnd";
-import { useContextMenu } from "react-contexify";
+import { theme, useContextMenu } from "react-contexify";
 import { useSelector } from "react-redux";
 import ReactTooltip from "react-tooltip";
-import { InfoCircleOutlined, WarningOutlined } from "@ant-design/icons";
+import { InfoCircleOutlined } from "@ant-design/icons";
```

Comment:
You should have a dark styling for the InfoCircleOutlined component, you can test the dark mode for this icon by adding the `ECON1101` course
i.e.
<img width="1014" alt="Screen Shot 2022-07-21 at 10 09 07 am" src="https://user-images.githubusercontent.com/65492346/180103145-1eb46d62-127f-4372-ae59-380bbfbeacc1.png">